cmd/ubuntu-image/main.go

```diff
@@ -0,0 +1,170 @@
+			fmt.Println("ubuntu-image snap functionality to be added later")
+		},
+	}
+
+	var Snap, Channel string
+	var ConsoleConf bool
+	snap.PersistentFlags().StringVarP(&Snap, "snap", "", "", snapFlagDesc)
+	snap.PersistentFlags().StringVarP(&Channel, "channel", "c", "", channelFlagDesc)
+	snap.PersistentFlags().BoolVarP(&ConsoleConf, "disable-console-conf", "", false, consoleConfFlagDesc)
+
+	// create the classic command and add classic specific flags to it
+	classic := &cobra.Command{
+		Use:   "classic [graph_tree]",
+		Short: "Create debian-based Ubuntu Classic image.",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Println("ubuntu-image classic functionality to be added later")
+		},
+	}
+
+	var Project, Filesystem, Suite, Architecture, Subproject, Subarchitecture, ExtraPpas string
+	var WithProposed bool
+
+	classic.PersistentFlags().StringVarP(&Project, "project", "p", "", projFlagDesc)
+	classic.PersistentFlags().StringVarP(&Filesystem, "filesystem", "f", "", filesystemFlagDesc)
+	classic.PersistentFlags().StringVarP(&Suite, "suite", "s", "", distributionFlagDesc)
+	classic.PersistentFlags().StringVarP(&Architecture, "arch", "a", "", architectureFlagDesc)
+	classic.PersistentFlags().StringVarP(&Subproject, "subproject", "", "", subprojFlagDesc)
+	classic.PersistentFlags().StringVarP(&Subarchitecture, "subarch", "", "", subarchFlagDesc)
+	classic.PersistentFlags().StringVarP(&ExtraPpas, "extra-ppas", "", "", ppasFlagDesc)
+	classic.PersistentFlags().BoolVarP(&WithProposed, "with-proposed", "", false, withProposedFlagDesc)
+
+	// add common flags to both commands
+	var Size, FileList, CloudInit, HooksDirectory, DiskInfo, Output string
+	var Debug bool
+	snap.PersistentFlags().StringVarP(&Size, "image-size", "i", "", sizeFlagDesc)
+	snap.PersistentFlags().StringVarP(&FileList, "image-file-list", "", "", fileListFlagDesc)
+	snap.PersistentFlags().StringVarP(&CloudInit, "cloud-init", "", "", cloudInitFlagDesc)
+	snap.PersistentFlags().StringVarP(&HooksDirectory, "hooks-directory", "", "", hooksFlagDesc)
+	snap.PersistentFlags().StringVarP(&DiskInfo, "disk-info", "", "", diskInfoFlagDesc)
+	snap.PersistentFlags().StringVarP(&Output, "output-dir", "O", "", outputFlagDesc)
+	snap.PersistentFlags().BoolVarP(&Debug, "debug", "d", false, debugFlagDesc)
+
+	classic.PersistentFlags().StringVarP(&Size, "image-size", "i", "", sizeFlagDesc)
+	classic.PersistentFlags().StringVarP(&FileList, "image-file-list", "", "", fileListFlagDesc)
+	classic.PersistentFlags().StringVarP(&CloudInit, "cloud-init", "", "", cloudInitFlagDesc)
+	classic.PersistentFlags().StringVarP(&HooksDirectory, "hooks-directory", "", "", hooksFlagDesc)
+	classic.PersistentFlags().StringVarP(&DiskInfo, "disk-info", "", "", diskInfoFlagDesc)
+	classic.PersistentFlags().StringVarP(&Output, "output-dir", "O", "", outputFlagDesc)
+	classic.PersistentFlags().BoolVarP(&Debug, "debug", "d", false, debugFlagDesc)
```

Comment:
```suggestion
	
	var subcommands = []*cobra.Command{snap, classic};
	for _, subcommand := range subcommands {
		subcommand.PersistentFlags().StringVarP(&Size, "image-size", "i", "", sizeFlagDesc)
        	subcommand.PersistentFlags().StringVarP(&FileList, "image-file-list", "", "", fileListFlagDesc)
		subcommand.PersistentFlags().StringVarP(&CloudInit, "cloud-init", "", "", cloudInitFlagDesc)
		subcommand.PersistentFlags().StringVarP(&HooksDirectory, "hooks-directory", "", "", hooksFlagDesc)
		subcommand.PersistentFlags().StringVarP(&DiskInfo, "disk-info", "", "", diskInfoFlagDesc)
		subcommand.PersistentFlags().StringVarP(&Output, "output-dir", "O", "", outputFlagDesc)
		subcommand.PersistentFlags().BoolVarP(&Debug, "debug", "d", false, debugFlagDesc)
	}
```

Maybe something like this for now? I'm no Go expert, but this seems to work.